### DISCUSSION TITLE
Prusa MK4 and Prusa Slicer STL files grayed out when trying to upload

### DISCUSSION THREAD

**Discussion by MaryMCook4** [community/community#119864] [Programming Help]
### Select Topic Area

Question

### Body

Hello, I am in prusa slicer and I’m trying to open a STL project from my files folder on my Mac but it won’t open a file because it is grayed out. Does anyone have a solution to this?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @MaryMCook4 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!

**Comment by iqnite** [comment]
There's an important difference between *files* and *projects*:
*Files* are single 3D models, like STL files. They only represent a 3D object and do not have any other properties like printer settings, infill, etc.
*Projects*, on the other hand, are a bundle of multiple 3D objects and configurations, and in PrusaSlicer they have the 3MF file extension. They contain printer settings and object placements and are ready to be sliced into GCODEs.

You can't open the STL files because you're doing that via *File > Open project*. Instead, click on *insert* in the toolbar (or press Ctrl+I).

Let me know if this worked!
